Mars-QA/Pages/Delete.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Mars_QA.Utilis;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mars_QA.Pages
+{
+    public class Delete : CommonDriver
```

Comment:
also your pages should be split up into areas for example langauge page, skill page and within these pages you should have the delete add and update functionality!